godot-core/src/builtin/vector3i.rs

```diff
@@ -0,0 +1,115 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use std::fmt;
+
+use godot_ffi as sys;
+use sys::{ffi_methods, GodotFfi};
+
+use crate::builtin::Vector3;
+
+/// Vector used for 3D math using integer coordinates.
+///
+/// 3-element structure that can be used to represent positions in 3D space or any other pair of
+/// numeric values.
```

Comment:
"any other pair of numeric values" is a bit imprecise; also, it's a triple

Reply:
Copy/paste mistake, sorry, now "triple". Replaced "numeric" by "floating-point" or "integer" as applicable.